app/controllers/avo/associations_controller.rb

```diff
@@ -190,5 +194,42 @@ def has_many_reflection?
         ActiveRecord::Reflection::HasAndBelongsToManyReflection
       ]
     end
+
+    def through_reflection?
+      @reflection.instance_of? ActiveRecord::Reflection::ThroughReflection
+    end
+
+    def additional_params?
+      additional_params.keys.count >= 1
+    end
+
+    def additional_params
+      params[:fields].except("related_id")
```

Comment:
Is this ok? Taking all the params that the user gives us?
It feels like a security issue. A bad actor can add fields in the DOM.

Can we just allow the fields that we need and know that the user added from the field DSL?